Source/Core/Core/PowerPC/BreakPoints.h

```diff
@@ -52,6 +54,7 @@ class BreakPoints
   using TBreakPointsStr = std::vector<std::string>;
 
   const TBreakPoints& GetBreakPoints() const { return m_breakpoints; }
+  TBreakPoints& GetBreakPoints() { return m_breakpoints; }
```

Comment:
I'm pretty sure that's unwanted as the `const reference` is here to prevent changes outside of the exposed interface. It tends to avoid undefined behaviours like holding to a pointer that can lead to crash later if the structure has changed. You should change the interface to expose the function you need (especially if they are only useful for GUI) like `SetBreakPointEnabled` or along those lines.